docs/source/user/threading-layer.rst

```diff
@@ -59,6 +59,21 @@ choosing a threading layer, the first is by selecting a threading layer that is
 safe under various forms of parallel execution, the second is through explicit
 selection via the threading layer name (e.g. ``tbb``).
 
+Setting the threading layer priority
+------------------------------------
+
+By default, The threading layers are searched in the order of
```

Comment:
```suggestion
By default, the threading layers are searched in the order of
```

Reply:
GitHub marked this as outdated. You already made a suggestion in b11cf88d0e5581db9436bcda8ae875d29cedf0c3.